modules/launch_utils.py

```diff
@@ -399,7 +399,14 @@ def prepare_environment():
         startup_timer.record("install open_clip")
 
     if (not is_installed("xformers") or args.reinstall_xformers) and args.xformers:
-        run_pip(f"install -U -I --no-deps {xformers_package}", "xformers")
+        try:

```

Comment:
I'm not familiar with rocm or AMD GPUs but feels like this should be done in webui.sh via `XFORMERS_PACKAGE` environment variable when not configured and trying to run `rocminfo`

similer to `TORCH_COMMAND`
https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/0120768f63dda88192576ee88c63d7f73eefb49a/webui.sh#L130-L175